Makefile.work

```diff
@@ -527,7 +530,8 @@ endif
 
 	$(Q)$(DOCKER_RUN) \
 		$(SLAVE_IMAGE):$(SLAVE_TAG) \
-		bash -c "$(SONIC_BUILD_INSTRUCTION) $@;$(COLLECT_BUILD_VERSION); $(SLAVE_SHELL)"
+		bash -c "$(SONIC_BUILD_INSTRUCTION) $@ && $(COLLECT_BUILD_VERSION) && $(SLAVE_SHELL)" \
+		|| $(ERROR_EXIT)
```

Comment:
I just realized, running the slave shell here (in `$(ERROR_EXIT)`) doesn't make sense, because the `docker run` command has already finished, and this will just launch the shell outside of the docker environment.

Reply:
done